video/out/vo_gpu_next.c

```diff
@@ -920,6 +920,12 @@ static bool draw_frame(struct vo *vo, struct vo_frame *frame)
     if (frame->still)
         params.frame_mixer = NULL;
 
+    if (frame->current->params.vflip) {
```

Comment:
`frame->current` can be null when changing VO during runtime

Reply:
For this, do `if (frame->current && frame->current->params.vflip)` instead?

Reply:
Seems fine